css/parser/tests/fixture/esbuild/misc/-8o_H6sq86TDAHqF7YO0hg/span.rust-debug

```diff
@@ -20,31 +20,31 @@ error: SelectorList
  --> $DIR/tests/fixture/esbuild/misc/-8o_H6sq86TDAHqF7YO0hg/input.css:1:1
   |
 1 | \2d {}
-  | ^^^
+  | ^^^^
```

Comment:
Same for here, ` ` (whitespace) is part of escaped characters, it is not simple whitespace